Exiled.API/Features/Player.cs

```diff
@@ -175,8 +175,25 @@ private set
         /// <summary>
         /// Gets the player's ammo.
         /// </summary>
+        [Obsolete("Use AmmoBag property instead.")]
         public Dictionary<ItemType, ushort> Ammo => Inventory.UserInventory.ReserveAmmo;
```

Comment:
Can you keep this one since we can add Normal ItemType in the ReserveAmmo now it's can be used by plugin